AnimatedComponents/src/components/search-bar/SearchBarDemo.tsx

```diff
@@ -6,29 +6,11 @@ import SearchBar from './SearchBar'
 const SearchBarDemo = () => {
   const [searchValue, setSearchValue] = useState('')
   return (
-    <View style={{ marginTop: 20 }}>
-      <Text style={{ textAlign: 'center', fontSize: 20, fontWeight: '700', color: '#016FC3' }}>
-        Search Bar
-      </Text>
-      <View
-        style={{
-          backgroundColor: 'white',
-          alignItems: 'center',
-          flexDirection: 'row',
-          marginBottom: 20,
-          paddingVertical: 10,
-          paddingHorizontal: 20,
-          borderBottomWidth: 1,
-          gap: 20,
-        }}>
-        <Text style={{ fontSize: 40, fontWeight: '700', color: '#016FC3' }}>VS</Text>
-        <SearchBar
-          inputContainerStyle={{ color: 'white', width: '90%' }}
-          searchValue={searchValue}
-          handleInputChange={setSearchValue}
-        />
-      </View>
-    </View>
+    <SearchBar
```

Comment:
confirm these changes with aman ,i guess you have not correctly resolved the conflicts.

Reply:
I have confirmed with him. As we are creating, the headings in the main App.tsx file but Aman was doing it in the demo and this was clashing. So I removed that code part.